pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.usort]
+
+categories = [
+    "future",
+    "standard_library",
+    "third_party",
+    "pytorch",
+    "pytorch_derivatives",
+    "first_party",
+]
+
+# This flag is needed to group torchvision into 'pytorch_derivatives' rather than 'first_party'
+first_party_detection = false
+
+[tool.usort.known]
+
+pytorch = ["torch"]
+pytorch_derivatives = ["torchvision"]
```

Comment:
This might seem excessive now, but we will probably depend on `torchdata` (not yet released) in the near future.

Reply:
Do we need a new category name for those? It seems that we could add `torchdata` to what used to be `first_party`?

I feel like having `first_party` as the internal test utils below doesn't really fit the intended definition of the "first_party" category.

Reply:
It's a matter of style. By default `first_party` groups the auto-detected package including all relative imports. I like to separate relative from absolute imports, because the former usually indicates that something is either not publicly exposed or at a different point. `isort` has a separate category for `local`, whereas `usort` has not. 

I've asked about this in facebookexperimental/usort#30 and the current implementation was the suggested workaround.

1. Do we want to spit absolute and relative imports? If not, your suggestion is the way to go.
2. Can we live with this workaround or should we send a patch to `usort`?

Reply:
> Do we want to spit absolute and relative imports?

yes, I would think so. But I would assume that the relative imports would always be separate from the absolute ones, even within the "first_party" group?
So IIUC the only difference will be that there will be a blank line between these, which IMHO isn't necessary